Packs/CortexXDR/TestPlaybooks/Test_XDR_Playbook.yml

```diff
@@ -2002,33 +2005,14 @@ tasks:
       brand: ""
     nexttasks:
       '#none#':
-      - "25"
+      - "63"
     scriptarguments:
       endpoint_ids:
-        complex:
-          root: PaloAltoNetworksXDR.Endpoint
-          filters:
-          - - operator: isEqualString
-              left:
-                value:
-                  simple: PaloAltoNetworksXDR.Endpoint.agent_status
-                iscontext: true
-              right:
-                value:
-                  simple: CONNECTED
-          - - operator: isEqualString
-              left:
-                value:
-                  simple: PaloAltoNetworksXDR.Endpoint.agent_type
-                iscontext: true
-              right:
-                value:
-                  simple: Workstation
-          accessor: endpoint_id
+        simple: aeec6a2cc92e46fab3b6f621722e9916
```

Comment:
why did you switch to a hard-coded value?

Reply:
Because the last value didn't perform the 'if' statement that leads to the wrong code, and this one does.